docs/quickstart.mdx

```diff
@@ -34,8 +34,18 @@ Let's make it happen!
 We're going to use Hardhat, but Sapphire should be compatible with your dev
```

Comment:
Perhaps you mention here or above that this Quickstart uses TypeScript and that we assume you have Node.js installed. Users can check out at the end of this chapter the same example in other languages.